nautobot/dcim/tests/test_api.py

```diff
@@ -379,8 +402,17 @@ def setUpTestData(cls):
         }
 
     def get_deletable_object_pks(self):
-        """Only return objects without children so that the deletion count is as expected."""
-        return [loc.pk for loc in list(Location.objects.filter(children__isnull=True))[:3]]
+        """Create new location objects without children so that the deletion count is as expected."""
+        lt5 = LocationType.objects.create(name="Delete Type")
+        loc_1 = Location.objects.create(name="delete 1", location_type=lt5)
+        loc_2 = Location.objects.create(name="delete 2", location_type=lt5)
+        loc_3 = Location.objects.create(name="delete 3", location_type=lt5)
+        return [loc_1.pk, loc_2.pk, loc_3.pk]
+
+    def get_deletable_object(self):
+        """To get the correct delete object count, make sure we avoid a cascade deletion."""
+        lt5 = LocationType.objects.create(name="Closet")
+        return Location.objects.create(name="delete this", location_type=lt5)
```

Comment:
These can be removed after you merge in my changes from https://github.com/nautobot/nautobot/pull/2579
However you'll need to create 3 `LocationType` instances in your `setUpTestData` since all of the `LocationTypes` created by the factories are being used